integrations/openai/integration.definition.ts

```diff
@@ -1,15 +1,30 @@
 import { IntegrationDefinition, interfaces, z } from '@botpress/sdk'
-import { modelId } from 'src/schemas'
+import { languageModelId } from 'src/schemas'
 
 export default new IntegrationDefinition({
   name: 'openai',
-  version: '2.2.2',
+  version: '2.3.0',
   readme: 'hub.md',
   icon: 'icon.svg',
   entities: {
     modelRef: {
       schema: z.object({
-        id: modelId,
+        id: languageModelId,
+      }),
+    },
+    imageModelRef: {
+      schema: z.object({
+        id: z.string(),
+      }),
+    },
+    imageGenerationParams: {
+      schema: z.object({
+        id: z.string(), // TODO: remove
```

Comment:
**todo:** do as your comment says once this is merged and deployed https://github.com/botpress/skynet/pull/2153